RELEASE.md

```diff
@@ -0,0 +1,20 @@
+# v3.0 is released!
```

Comment:
@gregor-robinson did you mean to add the 3.0.1 release in this file too?

Reply:
I added a note about the fix in 3.0.1 as part of 3.0 https://github.com/CitrineInformatics/lolo/pull/217/files/8510e4e64d80d761812dabff12813ce02cb96aaa#diff-781a53e648f3df8d16a08ec083b04bf4R20 -- just trying to reduce clutter.